xwiki-platform-tools/xwiki-platform-tool-configuration-resources/src/main/resources/xwiki.properties.vm

```diff
@@ -671,6 +671,12 @@ extension.versioncheck.environment.enabled=$xwikiPropertiesEnvironmentVersionChe
 #-# The default is:
 # security.script.save.checkAuthor=true
 
+#-# [Since 13.7]
```

Comment:
to be updated since 13.7 is not going to contain this PR.

Reply:
Sure